pephub/routers/api/v1/namespace.py

```diff
@@ -185,3 +187,58 @@ async def create_pep(
                 },
                 status_code=202,
             )
+
+@namespace.post(
+    "/upload_raw",
```

Comment:
This feels like it could be more *RESTful*... I almost think it makes more sense to have everything run through `POST /namespaces/{namespace}/projects` and determine the logic based on the `Content-Type`:

```python
@namespace.post('/projects')
async def create_new_pep(request: Request):
    content_type = request.headers.get('Content-Type')
    
    if content_type == 'application/json':
        data = await request.json()
        # insert using raw logic
        response = {'message': 'Function X executed successfully'}
    elif content_type.startswith('multipart/form-data'):
        form = await request.form()
        # insert using file logic
        response = {'message': 'Function Y executed successfully'}
    else:
        response = {'message': 'Invalid Content-Type'}
    
    return JSONResponse(content=response)
```

Does that make sense? When I interact with APIs from other projects (GitHub, Docker, etc), I never see something that looks like `upload_raw`. I think endpoints are supposed to describe nouns, not verbs.

**Here is another idea**... Lets **only** accept `application/json` content types. Then, on UI (and for any other clients) we parse an uploaded file and send that to the server to run the processing for us. Thoughts on that? This also solves the issue of artifacts in the config files due to having to save files to disk before uploading them 😀

Reply:
I like your second Idea. It's actually great, because we won't process files then. Do you know how can we implement it?

Reply:
Well, in theory, on the browser, when a user uploads a file, we can just parse the JSON into a string, and the csv into a string, and then send it up to `pephub` to process.

Reply:
Similarly, `pephubclient` can use python to read the contents of a file, then upload the stringified `yaml` + `csv`

Reply:
Question is: do we want to implement it in this iteration

Reply:
### Meeting Notes March 27th, 2023
- It was decided that the file handling should live on the server.
- We should have two endpoints:
1. `/namespaces/{namespace}/projects/files`
2. `/namespaces/{namespace}/projects/json`

The `pephub` web UI will use `/files` and `pephubclient` will use `/json`